client/my-sites/checkout/src/lib/pending-page.ts

```diff
@@ -170,6 +178,7 @@ export function addUrlToPendingPageRedirect(
 	const successUrlObject = new URL( successUrlBase );
 	successUrlObject.searchParams.set( 'redirectTo', url );
 	successUrlObject.searchParams.set( 'receiptId', String( receiptId ) );
+	fromSiteSlug && successUrlObject.searchParams.set( 'from_site', fromSiteSlug );
```

Comment:
Using the `&&` pattern is fine inside expressions where it is necessary to be short, but I think here the code would be more readable with a full `if` statement.